templates/cpp-template-default/proj.android/gradle.properties

```diff
@@ -18,23 +18,28 @@
 # org.gradle.parallel=true
 
 # Android SDK version that will be used as the compile project
-PROP_COMPILE_SDK_VERSION=22
+PROP_COMPILE_SDK_VERSION=25
 
 # Android SDK version that will be used as the earliest version of android this application can run on
 PROP_MIN_SDK_VERSION=14
 
 # Android SDK version that will be used as the latest version of android this application has been tested on
 PROP_TARGET_SDK_VERSION=27
 
-# Android SDK version that NDK will compile to
-PROP_APP_PLATFORM=14
+# Android Build Tools version that will be used as the compile project
+PROP_BUILD_TOOLS_VERSION=27.0.1
```

Comment:
actually, it is modified by @newnon , i think add `PROP_BUILD_TOOLS_VERSION ` just set a variable to avoid set number `27.0.1` everywhere. As for delete `PROP_APP_PLATFORM `, I don't known the reason

Reply:
PROP_APP_PLATFORM is now used in gradle.properties, so can just add a new property, not delete PROP_APP_PLATFORM. They are different properties.

Reply:
thanks for explain that, I will make `PROP_APP_PLATFORM ` back, and then do another test